frontend/src/pages/AdminForbiddenError/AdminForbiddenErrorPage.tsx

```diff
@@ -57,13 +61,13 @@ export const AdminForbiddenErrorPage = ({
               textAlign="center"
             >
               <Text as="h2" textStyle="h2">
-                You do not have access to this page.
+                t('features["admin-form"].adminForbiddenErrorPage.title')
```

Comment:
`features` does not contain an `admin-form` property. Did you try running the application after changing the code?